lib/middleman-hashicorp/partials/_enterprise-alert.html.erb

```diff
@@ -0,0 +1,12 @@
+<notextile>
```

Comment:
I haven't run into any issues with this tag rendering the content within in general browser tests, but the worst case scenario would probably be related to styling, not the content. The marketing team is discussing an official browser support policy, but from those discussions it's very clear that the furthest back we will be supporting will not be pre-html5. Are there relevant browser support requirements outside of the marketing team?